.buildconfig.yml

```diff
@@ -111,6 +111,10 @@ projects:
     path: components/feature/readerview
     description: 'Feature implementation providing a Reader View WebExtension.'
     publish: true
+  feature-script:
```

Comment:
Meta: I'm not fully convinced this needs to be a separate component but I can't find an existing one that makes sense to have this there.

It might be worth having one that we can use to place this and other support features like `ServiceWorkerSupport` into together - they are "features" but support functionality.

Not sure what that component should be called in order to describe it and also avoid it from being a dumping ground. 🤔

Reply:
Should we open another ticket for adding the support features component and then add the changes from this PR in the new component?

Reply:
Yes, I think that's good way forward to avoid feature creep.

Reply:
Opened https://github.com/mozilla-mobile/android-components/issues/12853 for this scope.